mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/route/handlers/ResourceClusterRouteHandler.java

```diff
@@ -54,4 +56,9 @@ CompletionStage<GetResourceClusterScaleRulesResponse> createAllScaleRule(
 
     CompletionStage<GetResourceClusterScaleRulesResponse> getClusterScaleRules(
         GetResourceClusterScaleRulesRequest request);
+
+    CompletionStage<GetResourceClusterSkuSizesResponse> createClusterSkuSize(
```

Comment:
how is this being used? isn't adding a new skuSize meaning updating the RC spec?

Reply:
yeah. no need for this anymore.